dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -125,4 +128,18 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
             }
         }
     }
+
+    /**
+     * help text output
+     */
+    private void wesCommandSuggestions(String runId) {
+        String wesEntryType = abstractEntryClient.getEntryType().toString().toLowerCase();
+        out("Useful commands for working with WES runs:");
```

Comment:
Since you have the specific commands below, I would just delete this.

Reply:
done